pkg/clusteragent/admission/mutate/autoinstrumentation/language_versions.go

```diff
@@ -101,12 +101,17 @@ var supportedLanguages = []language{
 	python,
 	dotnet,
 	ruby,
+	php, // PHP only works with injection v2, no environment variables are set in any case
 }
 
 func (l language) isSupported() bool {
 	return slices.Contains(supportedLanguages, l)
 }
 
+func (l language) isEnabledByDefault() bool {
+	return l != "php"
+}
+
```

Comment:
PHP should not be enabled by default because it will remain in Preview when other languages will go GA, but it must be "opt-in".
So it is possible to enable PHP support with a configuration like:
```
datadog:
 apm:
   instrumentation:
      enabled: true
      libVersions:
         php: "1"
```